doc/nrf/gs_programming.rst

```diff
@@ -29,28 +29,29 @@ You can download it from the following links:
 
 1. Extract the downloaded package and run the file :file:`bin/emStudio`.
 
-#. Select **File -> Open nRF Connect SDK Project.**
+#. Select **File -> Open nRF Connect SDK Project**.
 
-    .. figure:: images/ses_open.JPG
+    .. figure:: images/ses_open.png
        :alt: Open nRF Connect SDK Project menu
 
        Open nRF Connect SDK Project menu
 
-#. To import a project into SES, you must specify the following paths:
+#. The first time you import an |NCS| project, SES will prompt you to set the paths to the Zephyr Base directory and the GNU ARM Embedded Toolchain.
+   Set the Zephyr Base directory to the full path to ``ncs\zephyr``.
+   The GNU ARM Embedded Toolchain directory is the directory where you installed the toolchain (for example, ``c:\gnuarmemb``).
 
-	- **Zephyr Base** - the location of your cloned Zephyr repository
-	- **CMakeLists.txt** - the location of the :file:`CMakeLists.txt` project file
-	  of the sample that you want to work with
+   If you want to change these settings later, click **Tools -> Options** and select the **nRF Connect** tab.
+#. To import a project into SES, you must specify the following information:
+
+	- **CMakeLists.txt** - the location of the :file:`CMakeLists.txt` project file of the sample that you want to work with
+	- **Board Directory** - the location of the board description of the board for which to build the project
+	- **Board Name** - the board name (select from the list that is populated based on the board directory)
 	- **Build Directory** - the folder in which to run the build
```

Comment:
should we put a note, that this field we be auto filled based upon the board name, but can be changed by user ?
As example, if user select `nrf9160_pca10090ns`, the build folder will be pre-filled with:
`<path-to-app>/build_nrf9160_pca10090ns`

Reply:
added